### ISSUE TITLE
Astral Sorcery Starlight Abundance Change

### ISSUE THREAD

**Issue by jyo9678** [DarkPacks/SkyFactory-4#2141]
## Context

 I believe Starlight Infusers (and all Astral Sorcery "workbenches") should be altered to work anywhere, not just where there's an abundance of starlight. My Starlight Infuser is not working, and this makes me sad. Doing some research into Astral Sorcery, I'm seeing that it's most likely because of an incorrect location; however, I can't use the fosic resonator to find good spots of starlight because there is no ground for starlight to gather on. I think that feature should be altered or even removed; maybe changed so that everywhere has the same level of abundance (provided you build high enough)?

## Config/Script Details (if desired)
 I don't know the script well enough to know what to change.

**Comment by jyo9678** [comment]
Of course, if this is already the case and my Starlight Infuser just refuses to work, I'll just sit here and cry, I suppose.

**Comment by MBCollector672** [comment]
You can get enough starlight by just building high enough for the infusers. I have three of them automatically quadrupling diamond/emerald ores in my current playthrough.

One thing that caught me off guard is that it sometimes eats a bucket of starlight you placed to complete the multiblock structure, which causes it to stop working until that block is filled again.

**Comment by MajorPoison** [comment]
the Fosic Resonator does work in the overworld. the clouds of blue, sparkly smoke you're looking for will render on the highest block available, which for this pack is y=0. that means, if you wanna find a good patch of cloudy gas, get on some kind of flying armor and search at y=0, then when you find a spot, grab a Floaty block, go to y>120(mine are around 152, a Twilight forest beanstalk grown from y=0 provides a non-flight option to those heights) and place it down midair. if the block is in the cloud of gas, you should see particles rendering on top of it.
